server/src/scoring/handlers/userHandlers.ts

```diff
@@ -0,0 +1,27 @@
+﻿import { RequestHandler } from 'express';
```

Comment:
Jeżeli plik zawiera jedną funkcje to jestem zwolennikiem by nazywał się tak samo jak ta funkcja ale nie będę się kłócił